File: java/src/Response.java

class Response {
	private WikiPage page;
	private String http_code;

	public Response(WikiPage page, String http_code) {
		this.http_code = http_code;
		this.page = page;
	}

	public Response(WikiPage page) {
		this(page, "200");
	}

	public WikiPage page() {
		return page;
	}
}

File: csharp/Response.cs
using NUnit.Framework;
using System;
using System.Collections.Generic;

namespace WikiSearch
{
	public class Response
	{
		internal readonly WikiPage Page;
		internal readonly string HttpCode;

		public Response(WikiPage page, string httpCode)
		{
			this.HttpCode = httpCode;
			this.Page = page;
		}

		public Response(WikiPage page) : this(page, "200")
		{
		}
	}
}

